sympy/printing/latex.py

```diff
@@ -2571,6 +2571,15 @@ def _print_primeomega(self, expr, exp=None):
         return r'\Omega\left(%s\right)' % self._print(expr.args[0])
 
     def emptyPrinter(self, expr):
+        # support any objects which implement the ipython hook
+        try:
+            repr_latex_method = getattr(type(expr), '_repr_latex_')
+        except AttributeError:
+            pass
+        else:
+            # _repr_latex_ returns text mode, but we return math mode.
+            return r"\text{" + repr_latex_method(expr) + "}"
```

Comment:
I think I need a latex expert to weigh in on whether this is enough - does this need to be `\parbox` or something instead of just `\text`? Or is this idea flawed anyway?

Reply:
Asked on tex.SE here: https://tex.stackexchange.com/questions/554403/missing-parenthesis-in-display-math-within-text-within-display-math

Reply:
@asmeurer Any thoughts on this? This is a blocker for this PR.